plugins/system_upgrade.py

```diff
@@ -568,6 +577,9 @@ def run_reboot(self):
         if not self.opts.tid[0] == "reboot":
             return
 
+        if self.opts.poweroff_after is not None:
```

Comment:
You could probably drop this line and always assign the value from `opts` to `state`, right?

Reply:
Yep, misunderstood the `store_true` action, fixed.